.changes/config.json

```
@@ -232,7 +232,7 @@
       "dependencies": ["tauri-bundler", "tauri-utils", "tauri-macos-sign"],
       "postversion": [
         "cargo check",
-        "cargo build --manifest-path ../../crates/tauri-schema-generator/Cargo.toml"
+        "cargo build --manifest-path ../crates/tauri-schema-generator/Cargo.toml"
```

```suggestion
        "cargo build --manifest-path ../tauri-schema-generator/Cargo.toml"
```
the CWD for this one is `crates/tauri-cli`